src/LendingPool.sol

```diff
@@ -186,6 +202,10 @@ contract LendingPool is LendingPoolGuardian, Creditor, DebtToken, InterestRateMo
         treasury = treasury_;
         accountFactory = accountFactory_;
         liquidator = liquidator_;
+        initiatorRewardWeight = 1;
+        penaltyWeight = 5;
+        // note: to discuss
+        closingRewardWeight = 1;
```

Comment:
We need more decimals as two

Reply:
Updated to 4 which allows 0,01% precision